onadata/apps/logger/management/commands/update_attachment_storage_bytes.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+# vim: ai ts=4 sts=4 et sw=4 fileencoding=utf-8
+# coding: utf-8
```

Comment:
My bad, I'm used to add it because I faced some issues a while ago with some IDE (Netbeans on Windows IIRC). It was relying on this declaration even with Python 3.  It's been more than 10 years now.